src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Alert coverage gaps
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+  - Alert coverage gaps
+metaDescription: Learn how alert coverage gaps in New Relic help you identify gaps in your alert system. 
+---
+import alertcoveragegaps from 'images/alert-coverage-gaps-homepage.png'
+
+import addanalert from 'images/add-an-alert.png'
+
+import createapolicyalertcoveragegaps from 'images/create-a-policy-coverage-gaps.png'
+
+Alert Coverage gaps highlight areas in your stack where you're missing observability coverage. In our Alerts and AI section [we've created a homepage](https://one.newrelic.com/nrai/detection-gaps/home) where you can see all your uncovered entities so you can assess what information you're team is lacking and address the knowledge gap. 
```

Comment:
i would imagine this likely should be sentence case, like most of our features, so more like 'Our alert coverage gaps feature highlights ....'

Reply:
thanks!